pages.de/common/duf.md

```diff
@@ -0,0 +1,20 @@
+# duf
+
+> Festplattennutzung/freie Verwendbarkeit.
+> Weitere Informationen: https://github.com/muesli/duf.
```

Comment:
```suggestion
> Weitere Informationen: <https://github.com/muesli/duf>.
```

Reply:
@3voltAmperes Can you sync your fork with the main and edit the [page](https://github.com/tldr-pages/tldr/blob/main/pages.de/common/ical.md) to apply this suggestion?

Reply:
Why the sync? is really needed?

Reply:
> Why the sync? is really needed?

 `ical` file was deleted in his fork, it was moved in main. Since we need to edit the existing file I suggested sync.